### PR TITLE
Spec the css structure

### PR DESCRIPTION
**Pull request by jdfreder**
diagram pending, for now:

```
.notebook
  .cell.code
    .cell-toolbar
    .input-container
      .prompt
      .input
    .pagers
      .pager
    .outputs
  .cell.text
    .cell-toolbar
    .input-container
      .prompt
      .input
    .rendered
  .cell-creator
```

### CODE REVIEW

**Comment by Carreau** [comment]
> .notebook
>   .cell.code
>     .cell-toolbar
>     .input-container
>       .prompt
>       .input
>     .pagers
>       .pager
>     .outputs
>   .cell.text
>     .cell-toolbar
>     .input-container
>       .prompt
>       .input
>     .rendered
>   .cell-creator

Can we auto generate that from a fake rendered DOM, and test that it's still the same ?

**Comment by rgbkrk** [comment]
We can test the literal structure using `enzyme`.

**Comment by jdfreder** [comment]
Love the idea to add a test, doing it now....

**Comment by Carreau** [comment]
Damn I like being useful while doing nothing.

**Comment by jdfreder** [comment]
Okay, I just added a test.  I only test the CSS structure of things that aren't affected by user state.  i.e. .cell-creator is not tested because the mouse needs to hover between the cells.  Testing these user stateful things would be more complicated.

**Comment by rgbkrk** [comment]
We're now missing the padding on outputs.

screen shot 2016-05-17 at 8 02 26 am

File: static/styles/main.css [hunk 1]
```diff
@@ -58,6 +57,16 @@ td {
 }
 
 /*
+  Notebook
+ */
+
+.notebook {
+    paddingTop: 10px;
+    paddingLeft: 10px;
+    paddingRight: 10px;
```
Comment (rgbkrk) [inline static/styles/main.css:66]: 👍 glad to see this out of inline style

**Pull request merged**